lib/web/app/redirect.go

```diff
@@ -73,8 +77,16 @@ const js = `
           body: JSON.stringify(data),
         }).then(response => {
           if (response.ok) {
-            // redirect to the root and remove current URL from history (back button)
-            window.location.replace('/');
+            try {
+              if (path.charAt(0) !== "/") {
+                throw "malformed url"
+              }
+              window.location.replace(url.origin + path);
```

Comment:
I wonder about doing this instead to avoid the string concatenation and automatically handle whatever the path is
```suggestion
              redirect = new URL(path, url.origin)
              window.location.replace(url.toString());
```

Reply:
Great suggestion!

Reply:
I had a typo in the suggestion, should be fixed now

Reply:
Updated. Didnt even notice the typo because I slightly changed the var name anyway :)